Material.Blazor/Components/Card/MBCard.razor.cs

```diff
@@ -81,7 +111,7 @@ private protected override async Task DestroyMcwComponent()
         {
             if (PrimaryAction != null)
             {
-                await JsRuntime.InvokeVoidAsync("MaterialBlazor.MBCard.destroy", PrimaryActionReference);
+                //await JsRuntime.InvokeVoidAsync("MaterialBlazor.MBCard.destroy", PrimaryActionReference);
```

Comment:
We do not call `DestroyMcwComponent` anymore, right? Doesn't that make this change obsolete?

Reply:
Leaving that whole function in place was a bug and I've removed it.